### DISCUSSION TITLE
very slow to download

### DISCUSSION THREAD

**Discussion by zzzgit** [coqui-ai/TTS#2490] [General]
**Support/tts/tts_models--en--ljspeech--tacotron2-DDC**

 It took me almost 2 hours to download.

**Comment by TigerTy9** [comment]
Try using a newer python, I found python version 3.8 was very slow, try python 3.10. **Also try updating pip with python.exe -m pip install --upgrade pip**

 py -3.10 -m venv coqui_env
Activate the Virtual Environment:
    On Windows:
         coqui_env\Scripts\activate

Ensure that the virtual environment uses the correct Python version by checking:
python --version

pip install TTS
